sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -787,6 +788,77 @@ aditof::Status CameraItof::requestFrame(aditof::Frame *frame,
     return Status::OK;
 }
 
+aditof::Status CameraItof::normalizeABdata(aditof::Frame *frame,
+                                           int normalizesteps) {
+
+    using namespace aditof;
+    uint16_t *ab_video_data;
+    Status status = Status::OK;
+
+    if (normalizesteps == 0) {
+        return status;
+    }
+
+    status = frame->getData("ab", &ab_video_data);
+
+    if (status != Status::OK) {
+        LOG(ERROR) << "Could not get frame data!";
+        return status;
+    }
+
+    if (!ab_video_data) {
+        LOG(ERROR) << "no memory allocated in frame";
+        return status;
+    }
+
+    aditof::FrameDataDetails frameAbDetails;
+    frameAbDetails.height = 0;
+    frameAbDetails.width = 0;
```

Comment:
These 2 lines can be deleted, these values will be overwritten by the next function call.